do/loadbalancers.go

```diff
@@ -255,11 +255,19 @@ func (l *loadbalancers) nodesToDropletIDs(nodes []*v1.Node) ([]int, error) {
 
 	var dropletIDs []int
 	for _, node := range nodes {
+	Loop:
 		for _, droplet := range droplets {
 			if node.Name == droplet.Name {
 				dropletIDs = append(dropletIDs, droplet.ID)
 				break
 			}
+			addresses, _ := nodeAddresses(&droplet)
```

Comment:
Could we handle this error here

Reply:
I think we should only log it for now or change the `nodeAddresses` function, right now a single node without `private network` would cause a error. Why do we require `private network` anyway? It should be optional.
https://github.com/digitalocean/digitalocean-cloud-controller-manager/blob/57618014f2a6248c220d15383bcbdd24a6cb7485/do/droplets.go#L75-L92

We do not handle error in https://github.com/digitalocean/digitalocean-cloud-controller-manager/blob/57618014f2a6248c220d15383bcbdd24a6cb7485/do/droplets.go#L206 either.

Reply:
That is very reasonable, just logging works for me :)